backend/api/Controllers/AccessRoleController.cs

```diff
@@ -61,9 +61,6 @@ [FromBody] CreateAccessRoleQuery accessRoleQuery
             logger.LogInformation("Creating new access role");
             try
             {
-                if (accessRoleQuery.AccessLevel == RoleAccessLevel.ADMIN)
-                    return Unauthorized("Cannot create admin roles using API endpoints");
```

Comment:
Why are we removing this? I think it is meant to stop users from creating access roles with admin access. In my opinion the GHAS issue is misunderstanding what we do. But I think returning Unauthorized is wrong, should be BadRequest I think